tests/integration/integration-test.py

```diff
@@ -32,37 +35,243 @@ def setup(cls, config_path, openapi_path):
         cls.openapi = parser.specification
 
     @classmethod
-    def cleanup(cls):
+    def tearDownClass(cls):
```

Comment:
@tsoliangwu0130 I think we could make this change in the skeleton as well. Since the method doesn't require any arguments, we can use `tearDownClass` so that the function will automatically run after the tests have finished/failed